fuzzbench/test_e2e/test_e2e_run.py

```diff
@@ -19,7 +19,17 @@
 
 import pytest
 import redis
-from rq.job import Job
+import rq
+
+from common import config_utils, yaml_utils
+from experiment.build import docker_images
+
+
+@pytest.fixture(scope='class')
+def config():
```

Comment:
nit: consider making this a bit more descriptive eg by calling it `experiment_config`

Reply:
There is already a `experiment_config` and `local_experiment_config` in experiment/conftest.py for old architecture.
Maybe we can leave it as it is and later refactor/unify all of these three to a place.
wdyt?

Reply:
as discussed offline, there's no conflict with the other file